syntax = "proto3";

message RunningDetails {
  float distance = 1;
  float time = 2;
  uint32 heart_rate = 3;
  uint32 minutes_per_mile = 4;
}

message RunningMessageResponse {
  string message = 1;
}

service RunningMessage {
  rpc GetRunningMessage(RunningDetails) returns (RunningMessageResponse) {}
}